deno.json

```diff
@@ -0,0 +1,26 @@
+{
+  "tasks": {
+    "dev": "deno run --allow-net --allow-read --allow-env=NODE_DEBUG,NO_COLOR,FORCE_COLOR,TERM --watch --no-check ./content/main.tsx --dev",
+    "test": "deno test --allow-net --allow-read --allow-env=NODE_DEBUG,NO_COLOR,FORCE_COLOR,TERM --no-check=remote",
+    "serve": "deno run --allow-net --allow-read --allow-env --no-check ./content/main.tsx"
```

Comment:
`--no-check` and `--no-check=remote` are not needed here and above in modern deno